server/src/main/java/io/crate/analyze/relations/FullQualifiedNameFieldProvider.java

```diff
@@ -88,6 +88,7 @@ public Symbol resolveField(QualifiedName qualifiedName, @Nullable List<String> p
         boolean schemaMatched = false;
         boolean tableNameMatched = false;
         Symbol lastField = null;
+        ColumnUnknownException firstColUnknownException = null;
```

Comment:
Isn't this the last given that the loop continues over the relations?

Reply:
Yes, thanks for catching.